.circleci/config.yml

```diff
@@ -352,12 +352,16 @@ jobs:
             (cd src/improvements/script && shellcheck -x *)
 
 workflows:
-  main:
+  monorepo_integration_test_scheduled:
```

Comment:
I would suggest running the full main CI every four hours, not just these tests. The rationale being that if something onchain changes and it impacts an ongoing task we can find out

Reply:
Okay, I consolidated the workflows so we have one single workflow again.